package.json

```diff
@@ -133,6 +134,8 @@
     "react-redux": "7.2.4",
     "react-router-dom": "5.2.0",
     "react-scripts": "^4.0.1",
+    "react-virtualized-auto-sizer": "^1.0.6",
+    "react-window": "^1.8.6",
```

Comment:
this package seems not no be maintained anymore, the last release was published over a year go

Reply:
@yagopv recently used https://github.com/petyosi/react-virtuoso for new tx builder app

Reply:
It is [recommended over `react-virtualized`](https://github.com/bvaughn/react-virtualized#a-word-about-react-window) (by the same dev).

Reply:
I've just switched to it. It allowed for some simplification.